changelog.d/9910.misc

```diff
@@ -0,0 +1 @@
+Improve performance after joining a large room when presence is enabled.
```

Comment:
.feature?

Reply:
Are performance improvements features? They're not new buttons per se.

Reply:
the line is obviously fuzzy, but to me the question is whether it is interesting to Synapse admins/users - in which case, it's a `.feature` - or just developers, in which case it's a `.misc`.